### ISSUE TITLE
Java Issue When Resolving Manually a mission in MekHQ

### ISSUE THREAD

**Issue by demster42** [MegaMek/mekhq#3966]
### Environment
What version of MekHQ does your issue apply to? 49.18
What operating system are you using?  Windows, Mac, or Linux? Windows 11
What java version are you using?  11.0.17

### Description
Java Issue created when Clicking Finish when resolving a mission manually and a Mega Mek Mul is uploaded to it
This happens every time we finish a mission.  

### Files

[mekhq.log](https://github.com/MegaMek/mekhq/files/14888425/mekhq.log)

[megamek.log](https://github.com/MegaMek/mekhq/files/14888443/megamek.log)
[BC_Legion30390831.cpnx.gz](https://github.com/MegaMek/mekhq/files/14888461/BC_Legion30390831.cpnx.gz)
[BC_Legion.zip](https://github.com/MegaMek/mekhq/files/14888529/BC_Legion.zip)
[BC_Legion.zip](https://github.com/MegaMek/mekhq/files/14888538/BC_Legion.zip)

**Comment by HammerGS** [comment]
A bunch of these. ```


mekhq.MekHQ.lambda$main$0(MekHQ.java:240) - Uncaught Exception Detected
java.lang.NullPointerException
	at megamek.common.Entity.setSecondaryFacing(Entity.java:2649)
	at megamek.common.Tank.setFacing(Tank.java:435)
	at mekhq.campaign.Campaign.clearGameData(Campaign.java:5666)
	at mekhq.campaign.ResolveScenarioTracker.resolveScenario(ResolveScenarioTracker.java:1510)
	at mekhq.gui.dialog.ResolveScenarioWizardDialog.finish(ResolveScenarioWizardDialog.java:1372)
	at mekhq.gui.dialog.ResolveScenarioWizardDialog.lambda$initComponents$12(ResolveScenarioWizardDialog.java:1035)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1967)
	at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2308)
	at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:405)
	at java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
	at java.desktop/
```

**Comment by AaronGullickson** [comment]
its this line:

```java
GamePhase phase = getGame().getPhase();
```

However, in the most recent HEAD, there is now a check for game being non-null directly before that:

```java
if (game != null) {
            // Only allow changing secondary facing if we haven't done so in a prior phase
            GamePhase phase = game.getPhase();
            if (phase != null) {
                // If we already twisted in an earlier phase, return out
                if (getAlreadyTwisted()) {
                    return;
                }
                if (phase.isOffboard() || phase.isFiring()) {
                    // Only Offboard and Firing phases could conceivably have later phases with twisting
                    setAlreadyTwisted(true);
                }
            }
        }
```

This check was not there in 49.18. So, I think this issue has already been resolved in coding changes since 49.18.
